packages/map-template/src/components/Map/Map.jsx

```diff
@@ -63,10 +64,14 @@ function Map({ apiKey, gmApiKey, mapboxAccessToken, venues, venueName, onLocatio
             if (venueToShow) {
                 setVenue(venueToShow, mapsIndoorsInstance).then(() => {
                     onVenueChangedOnMap(venueToShow);
+                    // Set the map zoom level if the property is provided.
```

Comment:
We are calling setVenue twice in this file. Could the logic be moved into the setVenue function?

Reply:
Moving the logic into the `setVenue` function doesn't work, the `setZoom` seems to not change anything on the map.

Reply:
Did you wait for `fitVenue` to resolve before calling `setZoom`?